Xamarin.PropertyEditing.Mac/PropertyTableDataSource.cs

```diff
@@ -3,34 +3,80 @@
 using System.Linq;
 using AppKit;
 using Foundation;
-using Xamarin.PropertyEditing.Mac.ViewModels;
 using Xamarin.PropertyEditing.ViewModels;
 
 namespace Xamarin.PropertyEditing.Mac
 {
-	public class PropertyTableDataSource : NSOutlineViewDataSource
+	internal class PropertyTableDataSource
+		: NSOutlineViewDataSource
 	{
-		internal MacPanelViewModel ViewModel { get; private set; }
```

Comment:
I don't understand why the MacPanelViewModel class has been removed. It encapsulated all the logic for GetChildrenCount, ItemExpandable, GetChildObject and made the code here tidier.